rmgpy/data/kinetics/family.py

```diff
@@ -1527,6 +1530,8 @@ def apply_recipe(self, reactant_structures, forward=True, unique=True, relabel_a
             # (families with charged substances), the charge of structures will be updated
             if isinstance(struct, Molecule):
                 struct.update(sort_atoms=not self.save_order)
+            elif isinstance(struct, Fragment):
+                struct.update()
```

Comment:
Quick question: the molecule version above passes a flag about sorting atoms. Do the fragment objects need this as well?

Reply:
no, Fragment does not have atom sorting (cutting label does not have element) for now